cmd/merlinagent/main.go

```diff
@@ -32,11 +32,11 @@ import (
 	"time"
 
 	"github.com/fatih/color"
-	"github.com/satori/go.uuid"
 	"golang.org/x/net/http2"
 
 	"github.com/ne0nd0g/merlin/pkg/agent"
 	"github.com/ne0nd0g/merlin/pkg/messages"
+	"github.com/satori/go.uuid"
```

Comment:
@ahhh thank you as well for your quick responses. Can you put the `github.com/satori/go.uuid` back on line 35? The intention is to keep the 3 part libraries grouped together. Also, is possible for you to change your pull request against the `dev` branch? I might be able to do it myself when accepting the merge request.

Reply:
done and done. going to close this one. the fast responses are because there are more pertinent features to add, this is simply a litmus test.